### ISSUE TITLE
Draft README of how this works

### ISSUE THREAD

**Issue by danielbachhuber** [danielbachhuber/solr-demo-redux#1]

**Comment by danielbachhuber** [comment]
@joshkoenig @ataylorme I've put together a first draft for the README: https://github.com/danielbachhuber/solr-demo-redux/blob/master/README.md

At this point, I'd love your feedback on:

* Does the document have the content you'd expect? What information is missing or would it be worth to expand upon?

**Comment by ataylorme** [comment]
@danielbachhuber when testing out the demo I am seeing faster query times with Solr disabled than with it enabled.

I think for more complex queries it might be the other way around. Maybe adding search parameters for date, author, etc. that may to `WP_Query` would help.

One of the things I liked about the other [react demo](http://rest-ajax-solr-power.pantheonsite.io/react/) was showing the `WP_Query` params.

Would love @joshkoenig's feedback.

**Comment by danielbachhuber** [comment]
> when testing out the demo I am seeing faster query times with Solr disabled than with it enabled.
> 
> I think for more complex queries it might be the other way around. Maybe adding search parameters for date, author, etc. that may to WP_Query would help.

For indexed queries (e.g. http://dev-solr-demo-redux.pantheonsite.io/), this is correct. For unindex queries (e.g. http://dev-solr-demo-redux.pantheonsite.io/?sponsor=Sloan%20Foundation), Solr is much faster.

@joshkoenig It appears we need to disable MySQL query cache again. What's the PHP call to do that?

> One of the things I liked about the other react demo was showing the `WP_Query` params.

Dig it. If we can get them included in the design comps, they'll be easy to add to the site.

**Comment by ataylorme** [comment]
> For unindex queries (e.g. http://dev-solr-demo-redux.pantheonsite.io/?sponsor=Sloan%20Foundation), Solr is much faster.

Right, but on the current demo selecting a sponsor isn't available. It would be cool if we could surface those options.

**Comment by danielbachhuber** [comment]
> Right, but on the current demo selecting a sponsor isn't available.

It's available from the individual book view, if the book has a sponsor:

![image](https://cloud.githubusercontent.com/assets/36432/22269952/678a9c00-e242-11e6-82df-f40c870fbc81.png)

> It would be cool if we could surface those options.

I agree. Consider what we have as a rough proof of concept. @joshkoenig is going to get design for this shortly.

**Comment by ataylorme** [comment]
Got it, thanks @danielbachhuber. I was just out of the loop on the current state of the demo and next steps.

re: `README.md` the steps look okay. Once I have a bigger chunk of time I can go through them and report back on any issues.
